README.rst

```diff
@@ -57,8 +57,8 @@ This project has adopted the `Microsoft Open Source Code of Conduct <https://ope
 For more information see the `Code of Conduct FAQ <https://opensource.microsoft.com/codeofconduct/faq/>`_ or contact `opencode@microsoft.com <mailto:opencode@microsoft.com>`_ with any additional questions or comments.
 
 
-.. |Build status| image:: https://dev.azure.com/maluuba/bistring/_apis/build/status/microsoft.bistring?branchName=master
-    :target: https://dev.azure.com/maluuba/bistring/_build/latest?definitionId=132&branchName=master
+.. |Build status| image:: https://github.com/microsoft/bistring/actions/workflows/ci.yml/badge.svg
+    :target: https://github.com/microsoft/bistring/actions/workflows/ci.yml
 .. |Documentation status| image:: https://readthedocs.org/projects/bistring/badge/?version=latest
```

Comment:
Do you know why the latest version of the redthedocs badge displays as failing?

Reply:
https://github.com/readthedocs/readthedocs-docker-images/issues/107#issuecomment-684021245

I'm gonna try to work around that next :)

Reply:
Ah so the "latest" docs https://bistring.readthedocs.io/en/latest/ is actually an old commit (https://github.com/microsoft/bistring/commit/b79007e9)? But the badge is referring to the actual latest build?

Reply:
Yeah